.lagoon.yml

```diff
@@ -4,13 +4,31 @@ project: dpl-cms-core
 
 ssh: 20.238.147.183:22
 api: https://api.lagoon.dplplat01.dpl.reload.dk/graphql
+environment_variables:
+  git_sha: 'true'
 
 tasks:
   post-rollout:
+    - run:
+        name: Create new GH deployment
+        command: |
+          GH_DEPLOYMENT=$(curl -L -X POST -H "Authorization: Bearer $GH_DEPLOYMENT_TOKEN" \
+          https://api.github.com/repos/danskernesdigitalebibliotek/dpl-cms/deployments \
+          -d "{\"ref\":\"$LAGOON_PR_HEAD_BRANCH\", \"environment\":\"$LAGOON_PR_HEAD_BRANCH\", \
+              \"description\":\"Triggered by Lagoon\", \"transient_environment\": true, \
+              \"auto_merge\": false, \"required_contexts\": []}")
```

Comment:
It would be nice if we could set a deployment status with a link to the deployment log.

This is supported in the current setup. I think it would be even better to have it now that progressing from a pending to a successful (or failed) deployment status is 100% depenent of the Lagoon deployment.